.github/workflows/maven.yml

```
@@ -16,15 +16,34 @@ on:
+        include:
+          - vaadin: '14'
+            java: '17'
+          - vaadin: '23'
+            java: '17'
+          - vaadin: '24'
+            java: '17'
+          - vaadin: '25'
+            java: '21'
+    name: Vaadin ${{ matrix.vaadin }} (JDK ${{ matrix.java }})
     steps:
     - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)